src/core/dto/social-feed.dto.ts

```diff
@@ -55,3 +55,24 @@ export class UpdatePostInput {
   @Field(() => Date, { nullable: true })
   updatedAt?: Date;
 }
+
+@InputType()
+export class CreateCommentInput {
+  @Field(() => String, { nullable: true })
+  content?: string;
+
+  @Field(() => Date, { nullable: true })
```

Comment:
I would remove `createdAt` and `updatedAt` since we don't need the front-end to tell us that, we will most likely do that at the database level.